### PR TITLE
First cut at an FAQ. Trying not to make it too formal.

### PR DESCRIPTION
**Pull request by dirkx**
Can you folks give this a think? Am not sure it is a good idea ? But also want to stem the flow of Q's a bit.

### CODE REVIEW

**Comment by ryanbnl** [comment]
It made me chuckle, but I'm not sure that some of the more paranoid types will take it well :)

**Comment by hvisser** [comment]
Has anybody verified the claim of building the software? I don't think it's accurate and I've tried in the past and couldn't get it to verify entirely (this is documented in an issue: https://github.com/minvws/nl-covid19-notification-app-android/issues/14#issuecomment-675677797 ). If we include this in a faq is must obviously be correct. For pulling APKs from Android devices, no jailbreaking is needed either.

**Comment by dirkx** [comment]
Well - the escrow folks do it carefully.

I've personally only done it for both our first releases. It was doable then; but I had to fiddle with one of the linker settings on iOS (something pod-ish) - which also may be as I was at that time testing the ARM cross compile which was not officially in Xcode. Will see if I can find some time tomorrow to check again.

**Comment by ryanbnl** [comment]
It doesn't help at all for the backend mind. I guess that we're covered by the security audits validating the infra (i.e. that we're actually stripping IPs) and the notary trusting our auditing endpoints are telling the truth (which they are, but that's easily faked).

**Comment by hvisser** [comment]
> Well - the escrow folks do it carefully.

The Escrow folks don't check the binaries on the phone, they check source references and binaries published in the store, at least for Android.

My point is that if we suggest to build the app/pull it from a device to verify this, then it should be really reproducible otherwise it won't help folks that want to do this much and might lead to more confusion. Currently we don't have these procedures documented and Escrow is an alternative way to verify that the thing we publish is actually the same as the source.

File: README.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ One of the primary reason for making all this open source is to build trust. To
 However that only goes so far. How does one know that the code shown here is the code that was uploaded into the Apple and Google playstore ?
 
 To answer part of this question - this project has arranged for a variant of source code escrow[2]. 
-In this process an *independent* expert party verifies the source code repository, the build process and the final build; and compares this with the actual deployments to the backend servers and to the app stores. A notary ("Landsadvocaat"[3]) then verifies that this independent party has done this work, provided complete documentation, takes a statement of the engineers involved in the build and then issues a formal declaration that the are no differences found.
+In this process an *independent* expert party verifies the source code repository, the build process and the final build; and compares this with the actual deployments to the backend servers and to the app stores. 
+
+A notary ("Landsadvocaat"[3]) then verifies that this independent party has done this work, provided complete documentation, takes a statement of the engineers involved in the build and then issues a formal declaration that the are no differences found.
```
Comment (ExtraGG) [inline README.md:12]:
```suggestion
A notary ("Landsadvocaat"[3]) then verifies that this independent party has done this work, provided complete documentation, takes a statement of the engineers involved in the build and then issues a formal declaration that there are no differences found.
```

**Pull request closed without merging**